evergreen.yml

```diff
@@ -154,7 +154,7 @@ functions:
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        local_file: mongo-tableau-connector/package/mongodb_jdbc.taco
+        local_file: mongo-tableau-connector/package/mongodb_jdbc-v0.0.0.taco
```

Comment:
The file is created with this default version now.

Reply:
The version is now included in the filename.  It depends on the plugin-version that's in `connector/manifest.xml`. 

The default value we have there is 0.0.0, but when it's a tagged build then that value gets set to the actual version ie `1.2.0`.  So there needs to be a little more logic to have the value here be `v0.0.0` by default but if tagged set it to `v$MONGO_TABLEAU_CONNECTOR_VERSION`

Version number when tagged is updated here:
https://github.com/mongodb/mongo-tableau-connector/blob/master/evergreen.yml#L148